src/utils/formatNumber.ts

```diff
@@ -33,6 +33,11 @@ export const formatWad = (amt?: BigNumberish, config?: FormatConfig) => {
   return formattedNum(fromWad(_amt), config)
 }
 
+// Strips string of all commas
+export const stripCommas = (string: string) => {
```

Comment:
**praise**: love this

Reply:
we may be able to get around needing this if we use `fromWad()` instead of `formatWad()`!